docs/2.Basics/CLI Command Reference.md

```diff
@@ -18,7 +18,7 @@ nav_order: 2
 | `--external-checks-dir EXTERNAL_CHECKS_DIR` | Directory for custom checks to be loaded. Can be repeated. |
 | `--external-checks-git EXTERNAL_CHECKS_GIT` | Github url of external checks to be added. \n you can specify a subdirectory after a double-slash //. \n cannot be used together with --external-checks-dir' |
 | `-l`, `--list` | List checks. |
-| `-o [{cli,cyclonedx,json,junitxml,github_failed_only,sarif}]`, `--output [{cli,cyclonedx,json,junitxml,github_failed_only,sarif}]` | Report output format. |
+| `-o {cli,cyclonedx,json,junitxml,github_failed_only,sarif}`, `--output {cli,cyclonedx,json,junitxml,github_failed_only,sarif}` | Report output format. Add multiple outputs by using the flag multiple times (`-o sarif -o cli`) |
```

Comment:
@tsmithv11 would you like to update here: https://github.com/bridgecrewio/checkov/blob/master/checkov/main.py#L303 too so it will appear in the help command?

Reply:
Sure! Updated